subsys/bluetooth/host/hci_core.c

```diff
@@ -1248,7 +1248,7 @@ static inline bool rpa_timeout_valid_check(void)
 #if defined(CONFIG_BT_PRIVACY)
 	/* Check if create conn timeout will happen before RPA timeout. */
 	return k_delayed_work_remaining_get(&bt_dev.rpa_update) >
-	       K_SECONDS(CONFIG_BT_CREATE_CONN_TIMEOUT);
+	       (10 * bt_dev.create_param.timeout);
```

Comment:
This should be `K_MSEC(10 * bt_dev.create_param.timeout)` or otherwise we have to immediately fix it again when we convert the Bluetooth subsystem to support `k_timeout_t`

Reply:
Oh, sorry - nevermind: I read it wrong (as if it was a parameter to some k_delayed_work API.